.github/actions/setup-spack-stack/action.yaml

```diff
@@ -198,32 +204,12 @@ runs:
         echo "        prefix: /opt/intel/oneapi" >> ${SPACK_ENV}/spack.yaml
       fi
 
-      # Same for qt@5 on macOS or on Linux w/ Intel
-      if [[ "$RUNNER_OS" == "macOS" ]]; then
-        echo "" >> ${SPACK_ENV}/spack.yaml
-        echo "  packages:" >> ${SPACK_ENV}/spack.yaml
-        echo "    qt:" >> ${SPACK_ENV}/spack.yaml
-        echo "      buildable: False" >> ${SPACK_ENV}/spack.yaml
-        echo "      externals:" >> ${SPACK_ENV}/spack.yaml
-        echo "      - spec: qt@5" >> ${SPACK_ENV}/spack.yaml
-        echo "        prefix: /usr/local/opt/qt@5" >> ${SPACK_ENV}/spack.yaml
-      #elif [[ "${{ inputs.compiler }}" == "intel"* ]]; then
-      #  echo "" >> ${SPACK_ENV}/spack.yaml
-      #  echo "  packages:" >> ${SPACK_ENV}/spack.yaml
-      #  echo "    qt:" >> ${SPACK_ENV}/spack.yaml
-      #  echo "      buildable: False" >> ${SPACK_ENV}/spack.yaml
-      #  echo "      externals:" >> ${SPACK_ENV}/spack.yaml
-      #  echo "      - spec: qt@5" >> ${SPACK_ENV}/spack.yaml
-      #  echo "        prefix: /usr" >> ${SPACK_ENV}/spack.yaml
-      fi
-
       # Spack external find is by default only looking for build-tools. Either
       # search for additional packages explicitly, or fall back to the old
       # behavior to find all external packages.
-      #spack external find 
-      #spack external find git-lfs openmpi mpich
-      spack external find --all
-
+      spack external find 
+      spack external find git-lfs openmpi mpich perl openssl python qt
```

Comment:
we should also try to find curl (installed via homebrew on macOS, because building it fails with gcc)

Reply:
I think that qt5 and curl are not in the default search path on macOS with brew, may have to prepend PATH=...path-to-curl-bin:...path-to-qt5-bin:$PATH to this command (or export PATH correctly for this step in action.yaml)

Reply:
Just going through the special handling we have in the Github and trying to reduce it if possible. Just wanted to see the errors myself. I ran into the wget/openssl thing (I think macOS just doesn't have the libopenssl-dev libraries). Also ran into the GCC/Curl thing. Do we require an external Curl library? Could also disable `dap` in NetCDF in macOS.default.

Reply:
JEDI needs dap. We require curl on macOS to be installed via homebrew (in the reference config). It would build with clang, but not with gcc.

Reply:
Figured we also need wget, because this doesn't build on macOS with gcc either. Currently creating a PR for `spack` to be able to detect it with `spack external find`.

Reply:
That's a good idea. I was just wondering thinking about that and why it wasn't a thing. Are there multiple verisons of wget like, gsed vs. BSD sed, find, etc?

Reply:
> That's a good idea. I was just wondering thinking about that and why it wasn't a thing. Are there multiple verisons of wget like, gsed vs. BSD sed, find, etc?

No, but there is no "native" version in macOS - therefore the need to install via homebrew. PR will be ready in a minute.